packages/docker-build/src/commands/build.ts

```diff
@@ -200,9 +207,11 @@ export default class DockerBuildCommand extends BaseCommand {
             );
           }
 
+          const buildCommand = this.buildKit ? ['buildx', 'build'] : ['build'];
+
           await execUtils.pipevp(
             'docker',
-            ['build', ...this.args, '-f', dockerFilePath, '.'],
+            [buildCommand, ...this.args, '-f', dockerFilePath, '.'],
```

Comment:
You have to spread the `buildCommand` array to keep the second argument a `string[]`.

```suggestion
            [...buildCommand, ...this.args, '-f', dockerFilePath, '.'],
```